qiime/metadata.py

```diff
@@ -14,12 +16,33 @@ def __init__(self, dataframe):
     def __repr__(self):
         return self._dataframe.to_json()
 
+    @staticmethod
+    def load(path):
+        try:
+            df = pd.read_csv(path, sep='\t', dtype=object).\
```

Comment:
I'm good with it for now, unless there are specific issues. This is just a first pass for alpha. We need to have a discussion about this file format in the (post-SciPy) future.